### PR TITLE
feat(Community Permissions): Add conflicts waring in `New permission` page

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes [#8738](https://github.com/status-im/status-desktop/issues/8738)

### What does the PR do

- It adds conflicts warning in `New permission` page. Mocked data.
- It adds new pages and updates figma files in storybook.

### Affected areas

`Community Permission / Create new permission`

### Screenshot of functionality (including design for comparison)

https://user-images.githubusercontent.com/97019400/214085887-ffb10816-2088-44b3-b6d7-c2e361d31acc.mov

### CODE REVIEW

File: ui/app/AppLayouts/Chat/stores/CommunitiesStore.qml [hunk 1]
```diff
@@ -8,7 +8,14 @@ import utils 1.0
 QtObject {
     id: root
 
-    property var permissionsModel: ListModel {} // Backend permissions list object model asignement. Please check the current expected data in qml defined in `createPermissions` method
+    property var permissionsModel: ListModel {} // Backend permissions list object model assignment. Please check the current expected data in qml defined in `createPermissions` method
+    property var permissionConflict: QtObject { // Backend conflicts object model assignment. Now mocked data.
+        property bool exist: false
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/stores/CommunitiesStore.qml:13]: "exists"?

**Review by caybro — APPROVED** [review]
Looks good, some remarks inline

File: storybook/pages/CommunityNewPermissionViewPage.qml [hunk 2]
```diff
@@ -15,6 +15,13 @@ Pane {
             readonly property var assetsModel: AssetsModel {}
             readonly property var collectiblesModel: CollectiblesModel {}
             readonly property var channelsModel: ChannelsModel {}
+            property var permissionConflict: QtObject {
+                property bool exist: true
+                property string holdings: "1 ETH"
+                property string permissions: "View and Post"
+                property string channels: "#general"
+
```
[the remark this answers is not in the document]
Comment (noeliaSD) [reply storybook/pages/CommunityNewPermissionViewPage.qml:23]: This is for `storybook`!!

**Review by alexandraB99 — APPROVED** [review]

**Review by alexjba — APPROVED** [review]
Nice! LGTM 👍

**Review by glitchminer — APPROVED** [review]
Tested, approved.

Note: This was a static check only against the storybook UI component PermissionContactWarningPanel. Further testing will be required later in app under different conditions such as

- different asset names
- long channel names
- permissions applied to multiple assets
- permissions applied to multiple channels
- different app widths/heights
- dark mode

**Pull request merged**